chutney/packaging/local-dev/src/main/resources/application-sso-auth.yml

```diff
@@ -0,0 +1,43 @@
+
+spring:
+  security:
+    oauth2:
+      client:
+        registration:
+          my-provider:
+            provider: my-provider
+            client-id: my-client
+            client-secret: my-client-secret
+            authorization-grant-type: authorization_code
+            redirect-uri: "https://localhost:4200/login/oauth2/code/{registrationId}"
```

Comment:
we are not sure that we run app via `npm start`. 
I prefer to use   `https://${server.http.interface}:${server.port}` defined in `application.yml` file